main.py

```
@@ -17,12 +17,13 @@
 # Configure CORS for frontend access
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=["*"],  # In production, restrict this to your frontend's domain
+    allow_origins=["http://localhost:3000"],  # Add your actual frontend domains
     allow_credentials=True,
     allow_methods=["*"],
     allow_headers=["*"],
 )
 
+multi_agent = MultiAgent()
```

Pylint (3.3.7)

[error] Abstract class 'MultiAgent' with abstract methods instantiated

(E0110)